src/Parser.zig

```diff
@@ -390,6 +394,7 @@ pub const ObjectFrame = struct {
 pub const ScriptImport = struct {
     function: Ast.Node.Index,
     globals: std.ArrayListUnmanaged(Global) = .{},
+    global_names: std.StringArrayHashMapUnmanaged(u24) = .empty,
```

Comment:
No need for an `Array` hash map, its slower and only useful if you wanna preserve the insert order